### DISCUSSION TITLE
Where is the admin panel ?

### DISCUSSION THREAD

**Discussion by RolandTi** [JulianPrieber/littlelink-custom#395] [Questions]
Hello
Dumb question, but I just installed littlelink-custom, and I'm now on an empty page without any link to the admin panel.

The [documentation ](https://docs.littlelink-custom.com/setup/getting-to-know-the-admin-panel/)doesn't provide this information (?) neither the readme. And there is no /admin/ folder in the root

Please help ? :-)

Thanks !

![Capture d'écran 2023-04-13 213422](https://user-images.githubusercontent.com/28786873/231865169-0bbb37ba-5d0c-4a87-aece-6a76a36ef6b6.png)

**Comment by JulianPrieber** [comment]
Without more context, I'm assuming you can visit your home page at: `example.com/home`. From there you'll find everything you need.

**Reply by altinturk** [reply]
Could you please include these type of FAQs in the nicely organized documentation pages?

**Reply by JulianPrieber** [reply]
If you wish to improve our documentation, you can do so here: https://github.com/LinkStackOrg/docs

**Comment by RolandTi** [comment]
Hello, thanks for your quick reply, 
the post-install page was the screenshot above and the Home link was just a link to the same page. I added `/home/ ` next to the url, and I was able to login. Done !

Ok, I understood, if you check during install "Set user page as Home Page" (I used Alba on the example above), you're stuck in a loop with the home link that is the user page link. So you can't login except if you now the /home/ thing.

Thanks :)
